cms/envs/acceptance.py

```diff
@@ -98,7 +98,7 @@ def seed():
 USE_I18N = True
 
 # Include the lettuce app for acceptance testing, including the 'harvest' django-admin command
-INSTALLED_APPS += ('lettuce.django',)
+INSTALLED_APPS += ('lettuce.django', 'django.contrib.staticfiles')
```

Comment:
Check out this PR for lettuce: https://github.com/gabrielfalcao/lettuce/commit/14d0ac53dd0cbfd8434def5034d278b1a9e821f4#diff-b439adeadb7dada66034d2706d5ec075R39

Before this change lettuce was always loaded StaticFilesHandler when it started the app. After this change it will only load StaticFilesHandler if django.contrib.staticfiles is an installed_app. This was causing issues with the loading of static assets during the lettuce tests (which would prevent the browser bundle from loading and no tests running).

Side note: This was only an issue in CMS tests. LMS actually has the ^static/. route loaded and did not need this change to work correctly. We may want to normalize this at some point.